docs/windows.md

```diff
@@ -439,7 +449,10 @@ steps. And then run the following commands in powershell.
 mkdir c:\k\antrea
 cd c:\k\antrea
 curl.exe -LO https://github.com/antrea-io/antrea/releases/download/<TAG>/Start-AntreaAgent.ps1
+# Run antrea-agent without kube-proxy
 # $KubeConfigPath is the path of kubeconfig file
+./Start-AntreaAgent.ps1 -kubeconfig $KubeConfigPath -StartKubeProxy $false
```

Comment:
Why shall we run twice with "Start-AntreaAgent.ps1"  ( line 454 and line 456 )

Reply:
To indicate the different command which enables kube-proxy.

Reply:
If it is meaning that two command examples with different option values, maybe move the second one (requiring kube-proxy) into commented lines?

Reply:
Updated.